hawc/apps/assessment/migrations/0041_assessmentvalue_studies_and_more.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 5.1 on 2024-09-04 18:41
```

Comment:
Would it be too much trouble to merge the other migration files into this one? Just to cut down on the number we're introducing.

Reply:
Good find Matt. I did some digging and that's confirmed:

https://docs.djangoproject.com/en/5.1/ref/migration-operations/#runpython

> On databases that do support DDL transactions (SQLite and PostgreSQL), RunPython operations do not have any transactions automatically added besides the transactions created for each migration. Thus, on PostgreSQL, for example, you should avoid combining schema changes and RunPython operations in the same migration or you may hit errors like OperationalError: cannot ALTER TABLE "mytable" because it has pending trigger events.

https://stackoverflow.com/a/66799446/906385